### ISSUE TITLE
Do not show ellipsis if entire text is less than number of elements to show?

### ISSUE THREAD

**Issue by Trevoke** [bebraw/react-ellipsify#12]
The text I have to display is highly variable; I can check its length and use this component or a regular div, but it would be so much nicer if this component just "did the right thing" :)

**Comment by bebraw** [comment]
Yeah. That sounds like sane behavior to me. Please PR. 👍

**Comment by Trevoke** [comment]
Hi,

My schedule will not allow me to work on this for at least fifteen days, and by then I will not be working on React / Redux anymore. While I might be able to contribute, I'm afraid you can't count on me to do this.

**Comment by bebraw** [comment]
No problem. I'll hack together something quick for these. Simple enough. :+1:

**Comment by stahlmanDesign** [comment]
@bebraw did you ever work on this solution?

**Comment by bebraw** [comment]
@stahlmanDesign Nope, too busy and I forgot the issue. If you want, I can get the project up to date so it's easy to work with.

**Comment by stahlmanDesign** [comment]
That would be great, I'll give it a shot

**Comment by bebraw** [comment]
@stahlmanDesign I updated the setup. Pretty big change but worth it.

On a related note, maybe it would be a good idea to drop state from the component and control it from outside. This would allow us to go stateless style (function) and keep the component even simpler.

**Comment by bebraw** [comment]
I ended up going stateless as that was a simple change to do. Easier code as well now.
